src/styles/blocks/_menu.scss

```diff
@@ -0,0 +1,64 @@
+.menu {
+  background-color: $blue;
+
```

Comment:
Checklist 17. In case there is not enough space for all the menu items in the menu, add `overflow-y: auto;` to menu.